test/arrays/diten/replaceArrayAccess/arrayAlias2.bad

```diff
@@ -1,3 +1,2 @@
 2 6 12 20
 2 6 12 20
-arrayAlias2.chpl:2: error: halt reached - array reference count is negative!
```

Comment:
@daviditen - you might want to revisit --replace-array-accesses-with-ref-temps since this test no longer fails for other reasons.